services/helper.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-plusplus */
+
+module.exports = {
+  name: (id, sellers) => {
+    let name = '';
+    for (let i = 0; i < sellers.length; i++) {
+      if (sellers[i].id === id) {
+        name = sellers[i].name;
+        break;
+      }
+    }
+    return name;
+  },
```

Comment:
This can go for other functions that are similar also. Why not  just return the seller right away if it matches and return an empty string when it makes it to the end?